src/main/java/com/malinatran/router/Logger.java

```diff
@@ -0,0 +1,29 @@
+package com.malinatran.router;
+
+import com.malinatran.request.Request;
+
+import java.util.ArrayList;
+
+public class Logger {
+
+    private ArrayList<String> loggedRequestLines;
+    private String requestLines;
```

Comment:
Why keep an instance variable of this if it's only being used in `getLoggedRequests`?